compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/BIRPackageSymbolEnter.java

```diff
@@ -1453,7 +1485,31 @@ public BType readType(int cpI) throws IOException {
                     }
                     int funcCount = inputStream.readInt();
                     for (int i = 0; i < funcCount; i++) {
```

Comment:
Shall we extract this out to a method?

Reply:
Refactored in 1b6b21ff9a8ad716b114fd76ad04f19dc70a57e8